### PR TITLE
Implement objectCreateIdentifierName

### PR DESCRIPTION
**Pull request by duncanspumpkin**

### CODE REVIEW

**Review by AaronVanGeffen — COMMENTED** [review]
Sorry for the delay in testing. Any suggestions for testing this? Particular objects, perhaps?

**Comment by duncanspumpkin** [comment]
Save a scenario that has custom objects without packing the objects. Then delete the custom objects. Then load the save. The error message generated will trigger this code

**Comment by AaronVanGeffen** [comment]
Could you rebase this?

**Review by AaronVanGeffen — APPROVED** [review]

File: src/OpenLoco/S5/S5.cpp [hunk 1]
```diff
@@ -483,20 +485,28 @@ namespace OpenLoco::S5
...
+            if (c != ' ')
+            {
+                *dst++ = c;
+            }
+        }
+        *dst++ = '/';
+        std::stringstream ss;
+        ss << std::uppercase << std::setfill('0') << std::hex << std::setw(8) << header.flags << std::setw(8) << header.checksum;
+        const auto flagsChecksum = ss.str();
+        strcat(dst, flagsChecksum.c_str());
```
[the remark this answers is not in the document]
Comment (duncanspumpkin) [reply src/OpenLoco/S5/S5.cpp:511]: tbh i really want to use std::format on this ideally.

**Pull request merged**